packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -75,6 +75,54 @@ export default class CommentsRepo extends AbstractRepo<DbComment> {
     `, [limit]);
   }
 
+  async getPopularPollComments (limit: number): Promise<(DbComment)[]> {
+    return await this.manyOrNone(`
+      SELECT *
+      FROM public."Comments" AS c
+      WHERE c."parentCommentId" = 'NtsPs9wcwrpeK6KYL'
+      ORDER BY c."baseScore" DESC
+      LIMIT $1
+    `, [limit]);
+  }
+
+  async getPopularPollCommentsWithUserVotes (userId:string, limit: number): Promise<(DbComment)[]> {
+    return await this.manyOrNone(`
+    SELECT *
+    FROM public."Comments" AS c
+    INNER JOIN public."Votes" AS v ON c._id = v."documentId"
+    WHERE
+      c."parentCommentId" = 'NtsPs9wcwrpeK6KYL'
```

Comment:
Especially if you're reusing this in multiple places, it shouldn't be hardcoded repeatedly. Really should go in a database variable, take you 2 mins to do that.

Reply:
I think a database variable is worse (substantially less visible), but I do think it should be factored out into a variable

Reply:
In-general I think we should avoid database variables in almost all circumstances.

Reply:
(My guess is Ruby meant query parameter, or something)

Reply:
I meant use a query parameter but get the value from a Database variable.

@Discordius I'm not sure what you mean by "less visible" for database variables.